### PR TITLE
Chore/sc 21219/fix nitpicks

### PR DESCRIPTION
**Pull request by jkongie**
[sc-21219]

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -214,10 +214,8 @@ func (cc *ClientConn) handleMessageRequest(r *message.Request) {
 		// Create a decoder function to unmarshal the message
 		dec := func(v interface{}) error {
 			err := UnmarshalProtoMessage(r.GetPayload(), v)
-			if err != nil {
-				return err
-			}
-			return nil
+
+			return err
```
Comment (makramkd) [inline client.go:218]: I believe this could just be `return UnmarshalProtoMessage(...)`.

File: server.go [hunk 2]
```diff
@@ -204,10 +204,8 @@ func (s *Server) handleMessageRequest(pubKey credentials.StaticSizedPublicKey, r
 		// Create a decoder function to unmarshal the message
 		dec := func(v interface{}) error {
 			err := UnmarshalProtoMessage(r.GetPayload(), v)
-			if err != nil {
-				return err
-			}
-			return nil
+
+			return err
```
Comment (makramkd) [inline server.go:208]: Similar to prior comment.

**Comment by jkongie** [comment]
Updated

**Review by makramkd — APPROVED** [review]
:shipit:

**Pull request merged**